### ISSUE TITLE
Separate vectors from generator/ go code

### ISSUE THREAD

**Issue by austinabell** [filecoin-project/test-vectors#86]
Would be nice if the json vectors are put into a submodule or seperate repo so that clients that use the vectors as a submodule do not need to pull in all repos recursively.

The main issue personally is that my go tools show errors in my editor (VSCode). The issue was first that because submodules are not pulled recursively by default, ffi was not pulled in, but even after doing that the makefile commands and go code linting still errors so I can't get rid of the errors easily.

Side benefit is also having to pull in a smaller repo as a submodule, but nothing is functionally wrong, so this can be low priority for sure :)

Thanks!

**Comment by raulk** [comment]
Yeah, absolutely! 💯  This is the goal. For now it's way easier for us to keep both things in the same repo, as we are actively developing the toolkit. But in 3-weeks time, I hope they're separated.

**Comment by alanshaw** [comment]
I made a quick PoC for this over here https://github.com/alanshaw/test-vectors-corpus

It has a github action that runs hourly to regenerate the corpus from [filecoin-project/test-vectors](https://github.com/filecoin-project/test-vectors) and commit changes (sends a PR).
